geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/main/scala/org/locationtech/geomesa/parquet/ParquetFileSystemStorage.scala

```diff
@@ -44,39 +48,59 @@ class ParquetFileSystemStorage(context: FileSystemContext, metadata: StorageMeta
   override protected def createReader(
       filter: Option[Filter],
       transform: Option[(String, SimpleFeatureType)]): FileSystemPathReader = {
-    import scala.collection.JavaConversions._
 
     val filt = filter.getOrElse(Filter.INCLUDE)
 
-    // parquetSft has all the fields needed for filtering and return
-    val parquetSft = transform.map { case (_, tsft) =>
-      val transforms = tsft.getAttributeDescriptors.map(_.getLocalName)
-      val filters = FilterHelper.propertyNames(filt, metadata.sft).filterNot(transforms.contains)
-      if (filters.isEmpty) { tsft } else {
-        val builder = new SimpleFeatureTypeBuilder()
-        builder.init(tsft)
-        filters.foreach(f => builder.add(metadata.sft.getDescriptor(f)))
-        val psft = builder.buildFeatureType()
-        psft.getUserData.putAll(metadata.sft.getUserData)
-        psft
-      }
-    }.getOrElse(metadata.sft)
-
-    // TODO GEOMESA-1954 move this filter conversion higher up in the chain
```

Comment:
Does this PR address this ticket?  We closed it as "won't do".

Reply:
yeah i think i closed it b/c we decided there wasn't really any reason to do it, so i just removed the TODO here.